modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -113,6 +115,7 @@ To create an API asset:
 If the file is a ZIP, the selected main file must be in the root directory of the ZIP file. If the file is not a ZIP or if the file is a ZIP file with only one main file, then the main file is selected automatically.
 +
 . (Optional) Select *Advanced* and edit the advanced settings: *GroupId*, *AssetId*, *Version*, and *API version*. Exchange generates the group ID, asset ID, and version for you, and you can change these values here. You can change an API asset's version (asset version) and API version separately. The advanced settings are most often used to change the asset version.
+. (Optional) Select an option for *Lifecycle state*. The default is *Stable*.
```

Comment:
I guess you want to maintain consistency with the peer list items, but going forward let's convert these (Optional) steps to "If you want to..." statements or "Optionally, do this...."